City-of-Pori/pori-kada
# PORI-X Added map configuration for place input map

To test:

1. Sync from production (get the poris google api key to local this way)
2. drush fra -y && drush cc all
3. Go to edit any place content and make sure the map on the bottom of the form is displaying Pori.
